integrationTests/sync/edgeCases_test.go

```diff
@@ -20,6 +21,8 @@ func TestSyncMetaNodeIsSyncingReceivedHigherRoundBlockFromShard(t *testing.T) {
 		t.Skip("this is not a short test")
 	}
 
+	_ = logger.SetLogLevel("*:DEBUG")
```

Comment:
might remove this as running all tests in debug mode will output a plethora of messages.

Reply:
done.